controllers/activegate/statefulset.go

```diff
@@ -38,6 +39,7 @@ const (
 	DTNetworkZone     = "DT_NETWORK_ZONE"
 	DTGroup           = "DT_GROUP"
 	DTInternalProxy   = "DT_INTERNAL_PROXY"
+	DTDeployMetadata  = "DT_DEPLOY_METADATA"
```

Comment:
The title of the PR says _DT_DEPLOYMENT_METADATA_, for code modules it is _DT_DEPLOYMENT_METADATA_ as well. Does ActiveGate really use a different one?